alfresco-integration/src/main/java/com/github/dynamicextensionsalfresco/osgi/JavaPackageScanner.java

```diff
@@ -98,7 +98,7 @@
 		} catch (final IOException e) {
 			if (logger.isWarnEnabled()) {
 				logger.warn("Error scanning Java packages. This will cause Dynamic Extensions framework not to work!",
-						e.getMessage(), e);
+						e);
```

Comment:
Nit: Can you put this on the previous line with the rest of the statement?

Reply:
Line is too long then...